src/test/java/org/opensearch/security/system_indices/SystemIndicesTests.java

```diff
@@ -58,10 +61,16 @@ public class SystemIndicesTests extends SingleClusterTest {
         allAccessUser
     );
 
+    private static final String extensionUser = "extensions_user";
+    private static final Header extensionUserHeader = encodeBasicHeader(extensionUser, allAccessUser);
+    private static final String extensionUserC = "extensions_user_c";
+    private static final Header extensionUserCHeader = encodeBasicHeader(extensionUserC, allAccessUser);
+
     private void setupSystemIndicesDisabledWithSsl() throws Exception {
 
         Settings systemIndexSettings = Settings.builder()
             .put(ConfigConstants.SECURITY_SYSTEM_INDICES_ENABLED_KEY, false)
+            .put(ConfigConstants.SECURITY_SYSTEM_INDICES_ADDITIONAL_CONTROL_ENABLED_KEY, false)
```

Comment:
Lets try to change as few of these existing tests as possible, revert the whole file, and make new tests that are specifically oriented towards the new scenarios you've added.

Otherwise it isn't clear if we are changing existing behavoir, which should be the same using the default settings.